cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -103,6 +103,31 @@
   end
 end
 
+When(/^I authenticate Alice with Accept-Encoding header "([^"]*)"(?: with plain text body "([^"]*)")?$/) do |header, body|
+  body ||= ":cucumber:user:alice_api_key"
+  authn_request(
+    url: "/authn/cucumber/alice/authenticate",
+    api_key: body,
+    encoding: header.downcase.include?("base64"),
+    can: false
+  )
+end
+
+When(/^I authenticate Alice "([^"]*)" times in "([^"]*)" threads with Accept-Encoding: base64 header?$/) do |requests_num, threads_num|
```

Comment:
these two methods can be merged into one. please see this for reference: https://github.com/cyberark/conjur/blob/master/cucumber/authenticators_azure/features/step_definitions/authn_azure_steps.rb#L22

Reply:
In the second case, the Accept-Encoding is not optional ...Still. will change

Reply:
CHanged